apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/entity/vo/consumer/ConsumerInfo.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2023 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.portal.entity.vo.consumer;
+
+/**
+ * @see com.ctrip.framework.apollo.openapi.entity.Consumer
+ * @see com.ctrip.framework.apollo.openapi.entity.ConsumerRole
+ */
+public class ConsumerInfo {
```

Comment:
Create a VO of consumer here, we can add more attribute about it, for example, createTime, updateTime